ci/run.sh

```diff
@@ -8,6 +8,14 @@ if [ "$1" = "aarch64-apple-darwin" ] ; then
 	export CARGO_TARGET_AARCH64_APPLE_DARWIN_RUNNER=echo
 fi
 
+# Remove directories that are excluded by Cargo.toml
```

Comment:
I'm not too happy about duplicating these. Perhaps a better option would be to `cargo package`, then untar that and use it as the starting point for CI?

Reply:
Hmm, the CI runs... read-only?

Reply:
Ah, I see, we set the flag in run-docker.sh

I think I've got the `cargo package` approach working, I'll push it as a follow-up commit.